train-simulation/environment/vibration_peak_detector/src/analyzer.py

```diff
@@ -137,7 +143,8 @@ async def map_timestamp(self, ts):
                 if map_status == "ts-too-new":
                     # no location data available. Let's wait
                     _logger.info("Wait for newer location data")
-                    await asyncio.sleep(2)
+                    await asyncio.sleep(1)
+                    elapsed_time_seconds += elapsed_time_seconds
```

Comment:
I think this should be `elapsed_time_seconds += 1`

Reply:
sure